docs/setup/prerequisites/runtime.md

```diff
@@ -283,3 +284,112 @@ Make sure no libvirt is running on the worker nodes.
     	vmruntime_vms
    	vmruntime_libvirt
    	vmruntime_virtlet
+
+
+## iSulad
+
+[iSulad](https://gitee.com/openeuler/iSulad) , written in C/C++, is a lightweight container engine that has the advantage of being light, fast and applicable to multiple hardware specifications and architecture. iSulad has a wide application prospect.
+
+Follow the [install guide]( https://gitee.com/openeuler/iSulad/blob/master/README.md) to install and configure iSulad.
+
+### Steps
+1. **Install iSulad:**
+
+   If you are installing on an OpenEuler system, you can directly install through yum by adding OpenEuler's yum repository:
+
+   ```bash
+   $ cat << EOF > /etc/yum.repos.d/openEuler.repo
+   [openEuler]
+   baseurl=https://repo.openeuler.org/openEuler-22.03-LTS/OS/\$basearch
+   enabled=1
+   EOF
+   ```
+
+   Install iSulad with yum:
+
+   ```bash
+   $ yum install -y iSulad
+   ```
+   If your system is Centos, you can install iSulad by running [scripts]( https://gitee.com/openeuler/iSulad/blob/master/docs/build_docs/guide/script/install_iSulad_on_Centos_7.sh).
+
+   ```bash
+   $ git clone https://gitee.com/openeuler/iSulad.git
+   $ cd iSulad/docs/build_docs/guide/script
+   $ sudo ./install_iSulad_on_Centos_7.sh
```

Comment:
There is a small question, does this script depend on the entire iSulad repo when executed? If it is not dependent, can we download this file directly instead of cloning the entire code base? We can download only install_iSulad_on_Centos_7.sh through this link https://gitee.com/openeuler/iSulad/raw/master/docs/build_docs/guide/script/install_iSulad_on_Centos_7.sh

Reply:
yes ,only depend on scprit